spyder/config/main.py

```diff
@@ -222,8 +222,8 @@
               'tab_always_indent': False,
               'intelligent_backspace': True,
               'automatic_completions': True,
-              'automatic_completions_after_chars': 3,
-              'automatic_completions_after_ms': 300,
+              'automatic_completions_after_chars': 1,
+              'automatic_completions_after_ms': 50,
```

Comment:
50ms is very short and would likely be triggered at every keystroke. I think 300ms was chosen because that gives time to the user to type another key, avoiding too many requests?